dl.py

```diff
@@ -1,13 +1,36 @@
 import sys
 from multiprocessing import Process
 from pytube import YouTube
+import ffmpeg
 
 
 def dl_vid(url):
     yt = YouTube(url)
-    print('downloading: ' + yt.title)
-    yt.streams.filter(progressive=True, file_extension='mp4').order_by('resolution').desc().first().download() 
-    print('finished: ' + yt.title)
+    title = yt.title
+    print('Downloading: ' + title)
+
+    # Check if text contains # symbol, if so replace as ffmpeg doesn't like #
+    if title.__contains__("#"):
+        title = title.replace("#", "")
+        print("AFTER: " + title)
+
+    # trim to replace whitespace with underscore
+    title_trim = "_".join(title.split())
+
+    yt.streams.filter(progressive=True, file_extension='mp4')\
```

Comment:
i literally just read somewhere that you can enclose this statement in brackets so you can have them all on new lines. something like 
```
(yt.streams.filter(progressive=True, file_extension='mp4')
     .order_by('resolution')
     .desc())
```
change to that so we dont have to have the backslashes